src/streams.c

```diff
@@ -98,6 +98,59 @@ Int READ_COMMAND ( void )
     return 1;
 }
 
+Obj FuncREAD_ALL_COMMANDS( Obj self, Obj stream, Obj echo )
```

Comment:
Could you insert at least a minimal doc comment? Just 1-2 lines. And perhaps mention that this is used by the Jupyter GAP interface. Otherwise, a future code cleanup might decide to discard this function, as nothing uses it... ;-)..